tools/check_for_datafiles.py

```diff
@@ -17,7 +17,7 @@
     "sqlite3",
     "orc",
 ]
-whitelist_folders = []  # starting from project root dir
+whitelist_folders = ["/venv"]  # starting from project root dir
```

Comment:
I'm just wondering why the '/venv' folder is whitelisted and both the 'cache' and 'node_modules' folders are not? (since all of them are excluded directories inside the `tools/find_forbidden_words_in_repo.sh` script.)

Reply:
You could also add the other two in here if they would throw false positives. However, none of them currently do, which is why it's not required to add them in here. I got lots of false positives due to site packages having csv files and therefore appearing in the venv env.